test/new/db/cmd/feat_arroba

```diff
@@ -9,29 +9,29 @@ s @ 0x42
 EOF
 RUN
 
-NAME=@ e:
+NAME=@e:
```

Comment:
Having the parser accept `@     e:` is just additional complxity with no real benefit. The new parser just accepts `@e:` without any space.

Reply:
totally agree